optimade.rst

```diff
@@ -442,42 +442,62 @@ For example, the following query can be sent to API implementations `exmpl1` and
 
 :filter:`filter=_exmpl1_band_gap<2.0 OR _exmpl2_band_gap<2.5`
 
-Per property metadata fields
-----------------------------
-
-Implementations are allowed to specify an OPTIONAL field containing per entry metadata for a property.
-The name of the metadata field consists of the name of the property for which it contains the metadata, suffixed with "_meta".
-For example, when the field is :property:`cartesian_site_positions` the metadata field would be :field:`cartesian_site_positions_meta`.
+Metadata properties
+-------------------
 
-This metadata field consists of a dictionary which MAY contain database specific fields.
-If an implementation supports the metadata field, it SHOULD return the metadata field whenever the property to which the metadata field belongs is returned.
+A metadata property represents property-specific metadata for a given entry.
+How these are communicated in the response depends on the response format.
+For the JSON response format, the resource object metadata field, :field:`meta`, is used, see `JSON Response Schema: Common Fields`_.
 
-The metadata fields and their subfields should be described in the property definitions as described in the section `property definitions`_ just as regular fields.
-If a subfield is present in multiple metadata fields, these subfields should have a separate entry under each of these metadata fields.
-The subfields SHOULD have the same value for the :field:`$id` field if the :field:`$id` field is present and the subfields are otherwise identical.
-In addition to the fields described in the section `property definitions`_, the property definition of a metadata field SHOULD have the key:
+The metadata property is a dictionary of format specified by the field :field:`x-optimade-metadata-definition` in the Property Definition of the field, see `Property Definitions`_.
+Database providers are allowed to define their own metadata properties in :field:`x-optimade-metadata-definition`, but they MUST use the database-specific prefix even for metadata for database-specific fields.
```

Comment:
```suggestion
Database providers are allowed to define their own metadata properties in :field:`x-optimade-metadata-definition`, but they MUST use the database-specific prefix in the .
```
When a database adds their own fields, they always have to be prefixed by the database specific prefix. So I do not see why we need to specify this here.